src/System.ServiceModel.Tests.Common/src/BaseAddress.cs

```diff
@@ -5,6 +5,9 @@
 
 public static class BaseAddress
 {
+    // base address never used for end-to-end communication
+    public const string FakeServerBaseAddress = "http://localhost/fakeservice.svc";
```

Comment:
Line below says `// Base address for testing nonexistent endpoints` - now you've added another one, and that's getting kind of confusing. Could you either remove that comment (lines 12, 34), or update the comments so that the uses are clearly delineated?